src/Strategy/Python/Poetry.hs

```diff
@@ -71,14 +79,9 @@ findProjects = walk' $ \dir _ files -> do
         Just pbs ->
           if pbs == poetryBuildBackendIdentifier
             then pure ([project], WalkSkipAll)
-            else
-              context
-                ( "pyproject.toml does not use poetry build backend. It uses: "
-                    <> pbs
-                    <> "\n"
-                    <> poetryBuildBackendIdentifierHelpText
-                )
-                $ pure ([], WalkContinue)
+            else do
+              _ <- (warnIncorrectBuildBackend pbs)
```

Comment:
I tried using `$>` operator to simulate `as $> bs` ~ `do as; pure bs`. But ran into type issue.

Reply:
I had no problem, left a suggestion that built for me.